src/Calculator/Controls/EquationTextBox.cpp

```diff
@@ -421,6 +423,8 @@ bool EquationTextBox::RichEditHasContent()
 
 void EquationTextBox::OnRichEditMenuOpened(Object ^ /*sender*/, Object ^ /*args*/)
 {
+    VisualStateManager::GoToState(m_richEditBox, "Focused", false);
```

Comment:
Would be better to set m_HasFocus = true and call UpdateCommonVisualState() so that the visual state logic can be centralized there.